pkg/daemon/ceph/client/mirror.go

```diff
@@ -168,3 +168,26 @@ func GetPoolMirroringInfo(context *clusterd.Context, clusterInfo *ClusterInfo, p
 
 	return &poolMirroringInfo, nil
 }
+
+// EnableSnapshotSchedule configures the snapshots schedule on a mirrored pool
+func EnableSnapshotSchedule(context *clusterd.Context, clusterInfo *ClusterInfo, poolSpec cephv1.PoolSpec, poolName string) error {
+	logger.Infof("enabling snapshot scheduling for pool %q", poolName)
+
+	// Build command
+	args := []string{"mirror", "snapshot", "schedule", "--pool", poolName, poolSpec.Mirroring.SnapshotSchedule.Interval}
```

Comment:
Is a Ceph version check necessary for this command? Documenting the min Ceph version is probably fine if it matters.

Reply:
I'll add a check for Octopus min.